packages/integration-tests/projects/suite-web/run_tests.js

```diff
@@ -61,23 +59,6 @@ const wait = (timeout) => {
 }
 
 async function runTests() {
```

Comment:
Nit: all functions except of `runTests` and `getTestFiles` have arrow functions

Reply:
changed